### ISSUE TITLE
Auto-dispose axes when they are removed from xAxes/yAxes

### ISSUE THREAD

**Issue by martynasma** [amcharts/amcharts4#131]
It is extremely unlikely that removed Axis object will be reused. So it would make sense to auto-dispose it when "remove" event occurs on `xAxes` or `yAxes` lists.

Right now, people doing manipulation with axis need to take care of disposing of removed axes:

```
chart.yAxes.events.on("remove", function(ev) {
  if (!ev.oldValue.isDisposed()) {
    ev.oldValue.dispose();
  }
});
```

**Comment by zeroin** [comment]
added autoDispose property to Axis and Series with default values set to true

**Comment by martynasma** [comment]
FYI, `autoDispose` properties will be available in the next release.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.73](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.73).

## [4.0.0-beta.73] - 2018-11-07

### Changed
- `axisFills.template.interactionsEnabled` is now set to `false` by default. If you need to attach events on an axis fill, you will need to set it to `true`.

### Added
- `autoDispose` property added to `Series` and `Axis` classes with default value `true`. This means that series/axis will be automatically disposed when removed from chart's `series`, `xAxes`, or `yAxes` lists.
- It is now possible to have 3D stacked columns (by setting `clustered = false` on series).

### Fixed
- If chart container was moved in DOM tree, it stopped sensing resize and was not updating its size anymore.
- Sometimes `ValueAxis` could show zero label with a minus sign.
- Arrows on `MapLine` were not drawn.
- It was not possible to change the text color of the tooltip on `TreeMap` diagram.
- `axis.tooltipPosition = "pointer"` was not working as it should - always follow the mouse/pointer.
- If series had `data` set directly, it was still using chart's global `data`, resulting in combined datasets and potentially visual anomalies.
- Setting `cellStartLocation` did not have any effect on an already inited chart.
- Line series could be cut off in the beginning (when zoomed-in closely).
- Vertical axis ticks were not being positioned properly if axis had a title.
- 3D columns were not using chart's `depth`/`angle`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
